components/pages/create-bounty/controller.tsx

```diff
@@ -0,0 +1,723 @@
+import {useEffect, useState} from "react";
```

Comment:
can we break this file, at least, in file per step? at least will help to read this in the future :x